src/project.js

```diff
@@ -0,0 +1,177 @@
+import { v4 as uuidv4 } from 'uuid';
+import Task from './task';
+
+class Project {
+  constructor(name) {
+    this.name = name;
+    this.tasks = [];
+    this.head_id = uuidv4();
+    this.body_id = uuidv4();
+  }
+}
+
+const setDisplayNone = (name, date, priority, button) => {
+  name.style.display = 'none';
+  date.style.display = 'none';
+  priority.style.display = 'none';
+  button.style.display = 'none';
+};
+
+const storeProjectInLocalStorage = (project) => {
+  let projects;
+  if (localStorage.getItem('projects') === null) {
+    projects = [];
+  } else {
+    projects = JSON.parse(localStorage.getItem('projects'));
+  }
+  projects.push(project);
+  localStorage.setItem('projects', JSON.stringify(projects));
+};
+
+const storeTaskInLocalStorage = (task, project) => {
+  const projects = JSON.parse(localStorage.getItem('projects'));
+
+  projects.forEach((proj) => {
+    if (project.head_id === proj.head_id) {
+      proj.tasks.push(task);
+
+      localStorage.removeItem('projects');
+      localStorage.setItem('projects', JSON.stringify(projects));
+    }
+  });
+};
```

Comment:
- [ ] You seem to have added localstorage but it doesn't work correctly. After creating a project and todo item and refreshing the page the created items disappear. This shouldn't be the case with localStorage.